src/sections/migration/components/MigrationExportModal.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC } from "react"
+import { Button } from "components/Button/Button"
+import { Modal } from "components/Modal/Modal"
+import { Text } from "components/Typography/Text/Text"
+import {
+  MIGRATION_QUERY_PARAM,
+  MIGRATION_TARGET_DOMAIN,
+} from "sections/migration/MigrationProvider.utils"
+import MigrationLogo from "assets/icons/migration/MigrationLogo.svg?react"
+import { useTranslation } from "react-i18next"
+import { Separator } from "components/Separator/Separator"
+
+export const MigrationExportModal: FC<{
+  data: string
+  onCancel: () => void
+}> = ({ data, onCancel }) => {
+  const { t } = useTranslation()
+
+  return (
+    <Modal open headerVariant="FontOver">
```

Comment:
FontOver won't exist with changes styling

Reply:
I will update when we merge rebranding PR